### PR TITLE
Improve README.md and AUTHORS.txt

### PR DESCRIPTION
**Pull request by VictorLamoine**
A better version of https://github.com/PointCloudLibrary/pcl/pull/1083.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
-pcl
-===
+# Point Cloud Library
 
+![http://ns50.pointclouds.org/assets/images/contents/logos/pcl/pcl_horz_large_pos.png](PCL Logo)
+
```
Comment (SergioRAgostinho) [inline README.md]: I think this link is broken

Comment (VictorLamoine) [inline README.md]: Yup i'm fixing that and the image size right now!

**Comment by taketwo** [comment]
We can also have release number and licence icons like e.g. [there](https://github.com/IntelRealSense/librealsense/blob/master/readme.md).

**Comment by VictorLamoine** [comment]
There we go. Any idea or comment?

**Comment by SergioRAgostinho** [comment]
One comment: looks good :)

**Pull request merged**